docs/developer/eventing/sources/apiserversource/getting-started/README.md

```diff
@@ -30,119 +30,146 @@ command:
-      namespace: <namespace>
-    EOF
-    ```
-    Where:
+1. Create a ServiceAccount:
 
-    - `<service-account>` is the name of the ServiceAccount that you want to create.
-    - `<namespace>` is the namespace that you created in step 1 above.
-
-1. Create a ClusterRole by running the command:
-
-    ```yaml
-    kubectl create -f - <<EOF
-    apiVersion: rbac.authorization.k8s.io/v1
-    kind: ClusterRole
-    metadata:
-      name: <cluster-role>
-    rules:
-    - apiGroups:
-      - ""
-      resources:
-      - events
-      verbs:
-      - get
-      - list
-      - watch
-    EOF
-    ```
-    Where `<cluster-role>` is the name of the ClusterRole that you want to create.
-
-1. Create a ClusterRoleBinding by running the command:
-
-    ```yaml
-    kubectl create -f - <<EOF
-    apiVersion: rbac.authorization.k8s.io/v1
-    kind: ClusterRoleBinding
-    metadata:
-      name: <cluster-role-binding>
-    roleRef:
-      apiGroup: rbac.authorization.k8s.io
-      kind: ClusterRole
-      name: <cluster-role>
-    subjects:
-    - kind: ServiceAccount
-      name: <service-account>
-      namespace: <namespace>
-    EOF
-    ```
-    Where:
+    1. Create a YAML file using the template below:
 
-    - `<cluster-role-binding>` is the name of the ClusterRoleBinding that you want to create.
-    - `<cluster-role>` is the name of the ClusterRole that you created in step 3 above.
-    - `<service-account>` is the name of the ServiceAccount that you created in step 2 above.
-    - `<namespace>` is the name of the namespace that you created in step 1 above.
+        ```yaml
+        apiVersion: v1
+        kind: ServiceAccount
+        metadata:
+          name: <service-account>
+          namespace: <namespace>
+        ```
+        Where:
 
-1. Create the ApiServerSource object by running the command:
+        - `<service-account>` is the name of the ServiceAccount that you want to create.
+        - `<namespace>` is the namespace that you created in step 1 above.
 
-    === "kn"
+    1. Apply the YAML file by running the command:
 
         ```bash
-        kn source apiserver create <apiserversource> \
-          --namespace <namespace> \
-          --mode "Resource" \
-          --resource "Event:v1" \
-          --service-account <service-account> \
-          --sink <sink>
+        kubectl apply -f <filename>.yaml
         ```
-        Where:
+        Where `<filename>` is the name of the file you created in the previous step.
 
-        - `<apiserversource>` is the name of the source that you want to create.
-        - `<namespace>` is the name of the namespace that you created in step 1 above.
-        - `<service-account>` is the name of the ServiceAccount that you created in step 2 above.
-        - `<sink>` is the name of the PodSpecable object that you want to use as a sink.
-        A PodSpecable is an object that describes a PodSpec.
+1. Create a ClusterRole:
 
-    === "YAML"
+    1. Create a YAML file using the template below:
```

Comment:
Throughout, don't use "below". As per the Google developer style guide:

" below
    Don't use for a range of version numbers. Instead, use earlier.
    Don't use to refer to a position in a document. Instead, use later or following.
    Don't use to refer to a position in the UI. Instead, write instructions that avoid directional language. For more information, see Writing accessible documentation."

Maybe add this to the style guide also for easy reference, but it's in the [word list](https://developers.google.com/style/word-list?hl=en#letter-b)

Reply:
Sure, that's a reasonable suggestion. I'll make the changes and add this to the Knative word list.